tests/foreman/cli/test_contenthost.py

```diff
@@ -532,3 +553,280 @@ def test_bugzilla_1154611(self):
+            u'id': FAKE_0_ERRATA_ID,
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(result.stdout[0]['errata-id'], FAKE_0_ERRATA_ID)
+        self.assertEqual(result.stdout[0]['packages'], FAKE_0_CUSTOM_PACKAGE)
+
+    @run_only_on('sat')
+    def test_contenthost_apply_errata(self):
+        """@Test: Apply errata to content host
+
+        @Feature: Content Host - Errata
+
+        @Assert: Errata is scheduled for installation
+
+        """
+        # Create custom product and repository
+        custom_product = make_product({u'organization-id': self.org['id']})
+        custom_repo = make_repository({
+            u'url': FAKE_0_YUM_REPO,
+            u'content-type': 'yum',
+            u'product-id': custom_product['id'],
+        })
+        # Synchronize custom repository
+        result = Repository.synchronize({'id': custom_repo['id']})
+        self.assertEqual(result.return_code, 0)
+        # Associate repo with CV
+        result = ContentView.add_repository({
+            u'id': self.cv['id'],
+            u'repository-id': custom_repo['id'],
+            u'organization-id': self.org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        # Publish a new version of CV
+        result = ContentView.publish({u'id': self.cv['id']})
+        self.assertEqual(result.return_code, 0)
+        # Get the version id
+        result = ContentView.info({u'id': self.cv['id']})
+        self.assertEqual(result.return_code, 0)
+        cvv = result.stdout['versions'][-1]
+        # Promote version to next env
+        result = ContentView.version_promote({
+            u'id': cvv['id'],
+            u'to-lifecycle-environment-id': self.env['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        # List the subscriptions in given org
+        result = Subscription.list(
+            {u'organization-id': self.org['id']},
+            per_page=False
+        )
+        self.assertEqual(result.return_code, 0)
+        # Add subscription to activation-key
+        for subscription in result.stdout:
+            if subscription['name'] == custom_product['name']:
+                self.assertNotEqual(int(subscription['quantity']), 0)
+                result = ActivationKey.add_subscription({
+                    u'id': self.activation_key['id'],
+                    u'subscription-id': subscription['id'],
+                })
+                self.assertEqual(result.return_code, 0)
+        # Install custom package
+        result = self.vm.run(
+            'wget -nd -r -l1 --no-parent -A \'{0}.rpm\' {1}'
+            .format(FAKE_0_CUSTOM_PACKAGE, FAKE_0_YUM_REPO)
+        )
+        self.assertEqual(result.return_code, 0)
+        result = self.vm.run('rpm -i {0}.rpm'.format(FAKE_0_CUSTOM_PACKAGE))
+        self.assertEqual(result.return_code, 0)
+        # Apply errata to content host
+        result = ContentHost.errata_apply({
```

Comment:
I believe you are applying this in synchronous way.  Can you assert in the vm to make sure it got the errata?